### ISSUE TITLE
Cannot find an overload for "Execute" and the argument count: "1".

### ISSUE THREAD

**Issue by glaisne** [paulomarquesc/AzureRmStorageTable#20]
I've been trying to use this module with limited success. I know I've used it in the past without issue, but something is not working now. I did try both the 1.0.0.16 & 1.0.0.23 versions with the same results.

I'm trying to use the Add-StorageTableRow function and I've tried to boil everything down to the smallest pieces but without luck. I'm wondering if it is a .NET issue or something I'm missing.

My Error:
PS > $err |fl * -force

PSMessageDetails      :
Exception             : System.Management.Automation.MethodException: Cannot find an overload for "Execute" and the argument count: "1".
                           at CallSite.Target(Closure , CallSite , Object , Object )
                           at System.Management.Automation.Interpreter.DynamicInstruction`3.Run(InterpretedFrame frame)
                           at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
TargetObject          :
CategoryInfo          : NotSpecified: (:) [], MethodException
FullyQualifiedErrorId : MethodCountCouldNotFindBest
ErrorDetails          :
InvocationInfo        : System.Management.Automation.InvocationInfo
ScriptStackTrace      : at Add-StorageTableRow, C:\Program Files\WindowsPowerShell\Modules\AzureRmStorageTable\1.0.0.23\AzureRmStorageTableCoreHelper.psm1: line 191
                        at <ScriptBlock>, C:\Users\GLaisne\OneDrive - Carbonite\PowerShell\Carbonite\ProjectScripts\AzureVMPatchInfo\Examples\WrittingToStorageTable.ps1: line 139
                        at <ScriptBlock>, <No file>: line 1
PipelineIterationInfo : {}

PS > $err.invocationinfo

MyCommand             :
BoundParameters       : {}
UnboundArguments      : {}
ScriptLineNumber      : 191
OffsetInLine          : 11
HistoryId             : -1
ScriptName            : C:\Program Files\WindowsPowerShell\Modules\AzureRmStorageTable\1.0.0.23\AzureRmStorageTableCoreHelper.psm1
Line                  :                 return ($table.CloudTable.Execute((invoke-expression "[Microsoft.WindowsAzure.Storage.Table.TableOperation,$assemblySN]::insert(`$entity)")))

PositionMessage       : At C:\Program Files\WindowsPowerShell\Modules\AzureRmStorageTable\1.0.0.23\AzureRmStorageTableCoreHelper.psm1:191 char:11
                        + ...      return ($table.CloudTable.Execute((invoke-expression "[Microsoft ...
                        +                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PSScriptRoot          : C:\Program Files\WindowsPowerShell\Modules\AzureRmStorageTable\1.0.0.23
PSCommandPath         : C:\Program Files\WindowsPowerShell\Modules\AzureRmStorageTable\1.0.0.23\AzureRmStorageTableCoreHelper.psm1
InvocationName        :
PipelineLength        : 0
PipelinePosition      : 0
ExpectingInput        : False
CommandOrigin         : Internal
DisplayScriptPosition :

My Environment:
Version  Name
-------  ----
4.5.0    Azure.Storage
5.4.0    AzureRM.Profile
6.4.0    azurerm.resources
5.0.2    azurerm.storage
1.0.0.23 AzureRmStorageTable
1.0.0.16 AzureRmStorageTable
3.0.0.0  Microsoft.PowerShell.Host
3.1.0.0  Microsoft.PowerShell.Management
3.1.0.0  Microsoft.PowerShell.Utility
1.2      PSReadline
0.0      PSReadLineTheme

Name                           Value
----                           -----
PSVersion                      5.1.15063.1387
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.15063.1387
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

**Comment by glaisne** [comment]
I think I found the problem. It has to do with the way the modules are loaded.
I was starting with a shortcut which ran 'Add-AzureRMAccount.' That would cause the 'AzureRM' (4.4.0) module to load. Every time I would load the required modules, I would get an error, so I started running this after connecting to azure:
That sequence would result in no errors when importing the modules. I just tried a "virgin" prompt and imported the required modules in the order above and ran my script, and it worked perfectly.
